app/services/power_manager/bmc.rb

```diff
@@ -13,19 +13,39 @@ def ready?
 
     attr_reader :proxy
 
+    def power_action_v2
+      smart_proxy = host.smart_proxies.with_features(:BMC).first
+
+      return false unless smart_proxy
+      smart_proxy.has_capability?(:BMC, :poweraction_fix_3_17)
```

Comment:
```suggestion
      smart_proxy.has_capability?(:BMC, :power_action_v2)
```

We should rename the method in the smart proxy as well (https://github.com/theforeman/smart-proxy/pull/916 & https://github.com/theforeman/smart-proxy/pull/917)